library/psa_crypto_aead.h

```diff
@@ -148,4 +148,374 @@ psa_status_t mbedtls_psa_aead_decrypt(
     const uint8_t *ciphertext, size_t ciphertext_length,
     uint8_t *plaintext, size_t plaintext_size, size_t *plaintext_length );
 
+/** Set the key for a multipart authenticated encryption operation.
+ *
+ *  \note The signature of this function is that of a PSA driver
+ *       aead_encrypt_setup entry point. This function behaves as an
+ *       aead_encrypt_setup entry point as defined in the PSA driver interface
+ *       specification for transparent drivers.
+ *
+ * If an error occurs at any step after a call to
+ * mbedtls_psa_aead_encrypt_setup(), the operation is reset by the PSA core by a
+ * call to mbedtls_psa_aead_abort(). The PSA core may call
+ * mbedtls_psa_aead_abort() at any time after the operation has been
+ * initialized.
+ *
+ * After a successful call to mbedtls_psa_aead_encrypt_setup(), the PSA core
```

Comment:
We can probably remove this sentence or the one just above as they seem to say the same.

Reply:
I edited the first version as well as removing the second as I felt it didn't say precisely the same thing, but didn't really require another sentence.